tfe/client_mock.go

```diff
@@ -44,7 +44,7 @@ func (m *mockWorkspaces) Create(ctx context.Context, organization string, option
 	return ws, nil
 }
 
-func (m *mockWorkspaces) Read(ctx context.Context, organization string, workspace string) (*tfe.Workspace, error) {
+func (m *mockWorkspaces) Read(ctx context.Context, organization, workspace string) (*tfe.Workspace, error) {
```

Comment:
This change looks so curious to me! When I first look at it, I think that something is missing (the type specifier `string` next to organization). I'm wondering what others think about this change?

Reply:
this is the `paramTypeCombine` warning caught by `gocritic` linter, the suggestion is that whenever a fn has params of same type: `fn(a string, b string, c string)` => the shorter version is preferred:  `fn(a, b, c string)`. Some of the rules can be turned off if we prefer not to keep it

Reply:
We talked about this rule offline, and because this rule leads to more confusing code, we'd like to turn off the rule.